examples/linq/2.qpu_connection.ipynb

```diff
@@ -767,24 +767,24 @@
     }
    ],
    "source": [
-    "from tangelo.linq import Simulator\n",
+    "from tangelo.linq.simulator_base import SimulatorBase\n",
     "from tangelo.linq.helpers.circuits.measurement_basis import pauli_string_to_of\n",
     "\n",
     "# Rescale counts to obtain frequency histogram\n",
     "freqs = {k:v/100 for k,v in counts.items()}\n",
     "print(f\"Frequencies: {freqs}\")\n",
     "\n",
     "# Compute expectation value for ZZ operator\n",
-    "exp_ZZ = Simulator.get_expectation_value_from_frequencies_oneterm(pauli_string_to_of(\"ZZ\"), freqs)\n",
+    "exp_ZZ = SimulatorBase.get_expectation_value_from_frequencies_oneterm(pauli_string_to_of(\"ZZ\"), freqs)\n",
```

Comment:
Frustratingly obvious at this point (wish we realized it sooner): why are we importing a class if our goal is to run a static function that does not need anything from that class?

We should define `get_expectation_value_from_frequencies_oneterm` as a standalone function and import it directly (how about we import it with a smaller name btw? Like`expval_from_freqs_oneterm` or something, to use it anywhere a target simulator class is not needed.

Then we define a method in `SimulatorBase` that basically calls that function. As a result, the standalone function is available, but also usable through the children of the `SimulatorBase` classes.

We thus have no reasons to import `SimulatorBase`: we just import the function directly or we use an existing target simulator object returned by the `Simulator` function.